Sources/SharedServices/ExerciseService/ExerciseServiceStub.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  
+//
+//  Created by Anian Schleyer on 09.06.24.
+//
+
+import Common
+import SharedModels
+
+class ExerciseServiceStub: ExerciseService {
```

Comment:
Services for remote endpoints don't have state, this can be `struct`

Reply:
Good point, thanks. Will update this